### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by downsider**

### CODE REVIEW

File: src/TimeSkipper/NanoTimeSkipper.php [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Lexide\Chronos\TimeSkipper;
+
+class NanoTimeSkipper
```
Comment (UndeMa01) [inline src/TimeSkipper/NanoTimeSkipper.php]: Is this meant to be empty?

File: src/TimeSkipper/TimeSkipper.php [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Lexide\Chronos\TimeSkipper;
+
+class TimeSkipper implements TimeSkipperInterface
+{
+
+    public function skip(float|int $seconds): void
+    {
+        sleep((int) $seconds);
```
Comment (UndeMa01) [inline src/TimeSkipper/TimeSkipper.php:13]: We don't need the explict cast to int. Can also remove the float hint from the parameter.

Comment (UndeMa01) [reply src/TimeSkipper/TimeSkipper.php:13]: My bad I noticed its from an interface.

File: src/TimeSkipper/NanoTimeSkipper.php (hunk 1 above)
Comment (downsider) [reply src/TimeSkipper/NanoTimeSkipper.php]: No it is not

**Review by UndeMa01 — APPROVED** [review]

**Pull request merged**